assets/shared/structure/structure-store.js

```diff
@@ -28,149 +29,207 @@ export function registerStructureStore( {
 	storeName,
 	getEndpoint,
 	saveError,
+	fetchError,
 	clearError,
 	updateBlock,
 	readBlock,
+	...store
 } ) {
 	const DEFAULT_STATE = {
 		serverStructure: null,
+		editorStructure: null,
 		isSavingStructure: false,
-		hasStructureUpdate: false,
+		hasUnsavedServerUpdates: false,
+		hasUnsavedEditorChanges: false,
+		clientId: null,
 	};
 
 	const actions = {
 		/**
 		 * Fetch structure data from REST API.
 		 */
 		*loadStructure() {
-			const endpoint = yield* getEndpoint();
-			const result = yield apiFetch( {
-				path: `/sensei-internal/v1/${ endpoint }?context=edit`,
-			} );
-			yield actions.setStructure( result );
+			try {
+				const endpoint = yield* getEndpoint();
+				const result = yield apiFetch( {
+					path: `/sensei-internal/v1/${ endpoint }`,
+					method: 'GET',
+				} );
+				yield actions.setResult( result );
+			} catch ( error ) {
+				yield fetchError?.( error );
+			}
 		},
 
 		/**
 		 * Persist editor's structure to the REST API.
 		 *
```

Comment:
We have an extra line here that we can remove.

Reply:
Fixed in 3109a1a